File: 백준/Bronze/2588. 곱셈/곱셈.py
A = int(input())
B = int(input())
C = (int)((B % 100) % 10) * A
D = (int)((B % 100) / 10) * A
E = (int)(B / 100) * A
F = C + ((D * 10) + (E * 100))
print(C)
print(D)
print(E)
print(F)

File: 백준/Bronze/2588. 곱셈/곱셈.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
const a = parseInt(input[0]);
const b = parseInt(input[1]);

console.log((b % 10) * a);
console.log(Math.floor((b % 100) / 10) * a);
console.log(Math.floor(b / 100) * a);
console.log(a * b);